dockerfiles/coda_daemon_puppeteer.py

```diff
@@ -0,0 +1,105 @@
+# This is a temporary hack for the integration test framework to be able to stop
+# and start nodes dyamically in a kubernetes environment. This script takes
+# coda arguments and will start and monitor a coda process with those arguments.
+# If a SIGUSR1 signal is sent, it will stop this process, and if a SIGUSR2 is
+# sent, it will resume the process. Since this script is a hack, there are some
+# shortcomings of the script. Most notably:
+#   - the script will stack overflow after a lot of restarts are issued
+#   - the script does not attempt to handle errors from the tail child process
+
+import os
+from pathlib import Path
+import signal
+import subprocess
+import sys
+import time
+
+active_daemon_request = False
+inactive_daemon_request = False
+tail_process = None
+coda_process = None
+daemon_args = sys.argv[1:] if len(sys.argv) > 1 else []
+
+def escape_arg(arg):
+  return '"%s"' % arg
```

Comment:
This is currently unused. Should this be applied to `daemon_args`?

Reply:
No, it's a leftover from when I was using shell to do file redirection instead of configuring it directly. Removing.